webapp-django/crashstats/crashstats/models.py

```diff
@@ -826,6 +826,11 @@ def get(self):
             for x in result['hits']
         ]
 
+    def get_all(self):
+        """Return all platforms, not just the hardcoded ones.
```

Comment:
"not just the hardcoded ones" is not a true comment. 
It's all platforms without reducing it down to a list of dicts with the additional information whether it should be displayed or not.